### DISCUSSION TITLE
Are Material UI and panda CSS compatible?

### DISCUSSION THREAD

**Discussion by FatmaKahloun** [chakra-ui/panda#2639] [Q&A]
I'm creating my own components library and I want to use Material UI of React and Panda css to define my designes tokens and to customize the style of MUI components. But I find that Panda css has its own way to define style and designes tokens and MUYI also has its own way.
So I want to know is this two library are compatible and I can use them together or not ? 

Thank you in advance :smile:

**Comment by anubra266** [comment]
@FatmaKahloun This doc can answer all your questions in this regard.

You should be possible to use them together
https://mui.com/material-ui/integrations/interoperability/
